ibis/omniscidb/client.py

```diff
@@ -805,6 +805,8 @@ def _execute(
         Exception
             if execution method fails.
         """
+        # time context is not implemented for omniscidb yet
+        kwargs.pop('timecontext', None)
```

Comment:
Can we just not pass `timecontext` in kwargs in omniscidb instead of removing it here? e.g., if the user call `expr.execute()`, then don't put timecontext in kwargs.

Reply:
Add here just because it is the line of failing, I agree it's cleaner to remove `timecontext` if it is not present by user. Removed in latest commit.